### PR TITLE
Uirouter migration bug fix - Remove unused false path in logout and add the logout redirect

### PR DESCRIPTION
**Pull request by rowasc**
This pull request makes the following changes:
- Add the logout redirect to map
- Remove path that never happens 

Testing checklist:
- [ ] Logout
- [ ] Check that you are redirected to the map after logging out

Fixes ushahidi/platform# .

Ping @ushahidi/platform

### CODE REVIEW

File: app/common/auth/authentication-events.run.js [hunk 1]
```diff
@@ -42,17 +42,13 @@ function AuthenticationEvents($rootScope, $location, Authentication, Session, _,
...
         $rootScope.loggedin = false;
         // we don' wat to reload until after filters are correctly set with the backend default that the user
         // would get when logged out
         PostFilters.resetDefaults().then(function () {
-            if (redirect) {
-                $location.url(redirect);
-            }
-
-            $state.go($state.$current.name ? $state.$current : 'map', null, { reload: true });
+            $state.go('posts.map');
```
Comment (rjmackay) [inline app/common/auth/authentication-events.run.js]: I don't think this should be the case. If I'm on a post detail view and I log out... I should stay on the same view.

Comment (rowasc) [reply app/common/auth/authentication-events.run.js]: ok. 
This was reported as a requirement/bug so I thought it was the expected behavior. Will change it..

Comment (rjmackay) [reply app/common/auth/authentication-events.run.js]: Hrm. I might be wrong? Where was it logged/who was it logged by?

Comment (rowasc) [reply app/common/auth/authentication-events.run.js]: https://ushahidi.slack.com/archives/C03UU507R/p1510670345000387 Ah, ok I think Carolyn reported it to me in chat, and asked here. 
But we might be wrong?

**Comment by rjmackay** [comment]
Closing. This isn't the correct behaviour.
Currently we keep you on the same page and try to reload it. I think thats right, with the exception that if you were on say settings or a draft post you should get redirected to map.

**Pull request closed without merging**